### DISCUSSION TITLE
Welcome to python-verisure Discussions!

### DISCUSSION THREAD

**Discussion by persandstrom** [persandstrom/python-verisure#137] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by krambriw** [comment]
Hi, I'm Walter, been using the great python-verisure for years, works excellent! I have it integrated in my own designed Python script that in addition, amongst other things, holds support for MQTT and integration with my video security system (based on Motion).
My Home Automation system is a home brewed thing based on Node-RED where I integrate all kind of things, like Verisure and Shellies for light & power control, heat pump monitoring etc etc

Earlier days I developed & contributed many plugins in Python for Eventhost and that was the first place where I used Per's great contribution

Kind regards, Walter

![image](https://user-images.githubusercontent.com/20238676/185788411-0f1fa7c6-5fa3-4953-ab7a-e56ef5335104.png)

**Comment by niro1987** [comment]
Hi @persandstrom, I just followed [this blog](https://www.seanh.cc/2022/05/21/publishing-python-packages-from-github-actions/) to automatically publish new github releases to pypi on [my own project](https://github.com/niro1987/python-sapcommissions). I thought this might be something nice for you too.
